### ISSUE TITLE
Need to run from command line to pick up the kubectl context

### ISSUE THREAD

**Issue by robvadai** [DawidMyslak/kubernetes-helm-ui#5]
First of all, this app looks very promising and well done for building it!

The issue I'm facing is that the app has to be ran using `open "/Applications/Coolbernetes.app"` from the terminal otherwise it can't pick up the correct `kubectl` context.

**Comment by DawidMyslak** [comment]
Hey @robvadai, sorry for delayed response, I'm watching way too many repositories and my GitHub notifications are total mess, completely missed this one.

I just wonder if you used zip file or dmg installer? To be honest I've never experienced this issue on my Mac, also my colleagues from work that are using Coolbernetes didn't report it.

Any ideas what could cause this? I could try to release a patch with fix, but have no clue how to reproduce this problem.

**Comment by DawidMyslak** [comment]
Hey @robvadai did you manage to resolve this?

**Comment by robvadai** [comment]
Just getting this when I run the app (not from Terminal):

![screen shot 2018-06-18 at 11 02 29](https://user-images.githubusercontent.com/20482782/41530148-3540f26c-72e7-11e8-9926-29c7008128c3.png)

I'm using High Sierra version 10.13.4

**Comment by robvadai** [comment]
I used the DMG installer

**Comment by DawidMyslak** [comment]
Spinner is spinning forever or you're seeing an error after some time? Anything in the console?

**Comment by robvadai** [comment]
Yeah it's spinning forever. How can I get the logs when I don't start it from the terminal?

**Comment by DawidMyslak** [comment]
@robvadai Normally everything is available in the "Console" tab, but you can't access it obviously because of the spinner blocking the entire screen. You could try to build it yourself (https://github.com/DawidMyslak/kubernetes-helm-ui#build-it-yourself) and then everything should be visible in the Node.js terminal window.
